src/Kopernicus/Configuration/SpaceCenterLoader.cs

```diff
@@ -366,6 +366,10 @@ public ColorParser EditorGroundColorParser
             }
         }
 
+        [ParserTarget("GrassMaterial", AllowMerge = true)]
```

Comment:
I would prefer this to be just `Material` like it is in most if not all other cases in Kopernicus.

Reply:
I wanted to make a distinction from "SpaceCenter/Material" since this material only applies to the grass. however, I can change it to Material if you prefere

Reply:
this has been changed